src/Form/AdTechMediaConfigForm.php

```diff
@@ -56,216 +56,260 @@ public function getEditableConfigNames() {
   }
 
   /**
+   * Get a list of all configuration options.
+   *
+   * @return array
+   *   A list of configuration options.
+   */
+  private function getConfigurationOptions() {
+    return [
+      'api_key',
+      'country',
+      'revenue_model',
+      'email',
+      'content_pricing',
+      'content_currency',
+      'content_paywall',
+      'content_paywall_type',
+      'content_preview',
+      'content_preview_type',
+      'locking_algorithm',
+      'dns_access',
+      'social_media',
+      'pledge_view',
+      'ad_view',
+      'pay_view',
+      'refund_view',
+      'price_view',
+    ];
+  }
+
+  /**
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
     $config = $this->config('adtechmedia.settings');
 
-    $form['general'] = array(
+    $form['general'] = [
       '#type' => 'details',
       '#title' => $this->t('General configuration'),
       '#open' => TRUE,
-    );
+    ];
 
-    $form['general']['api_key'] = array(
+    $form['general']['api_key'] = [
       '#type' => 'textfield',
       '#title' => $this->t('API Key'),
       '#default_value' => $config->get('api_key'),
       '#required' => TRUE,
       '#title_display' => 'after',
       '#suffix' => '<span class="bar"></span>',
-    );
+    ];
 
-    $form['general']['regenerate'] = array(
+    $form['general']['regenerate'] = [
       '#type' => 'button',
       '#value' => $this->t('Regenerate'),
-      '#attributes' => array(
-        'class' => array('btn', 'regenerate'),
-      ),
-      '#ajax' => array(
-        'callback' => array($this, 'regenerateApiKeyCallback'),
+      '#attributes' => [
+        'class' => ['btn', 'regenerate'],
+      ],
+      '#ajax' => [
+        'callback' => [$this, 'regenerateApiKeyCallback'],
         'event' => 'click',
         'wrapper' => 'edit-api-key',
         'method' => 'replaceWith',
-      ),
-      // '#prefix' => '<i class="mdi mdi-autorenew"></i>',.
-    );
+      ],
+      '#limit_validation_errors' => [],
+    ];
 
-    $form['general']['country'] = array(
+    $form['general']['country'] = [
       '#type' => 'select',
       '#title' => $this->t('Country'),
-      '#options' => array(
+      '#options' => [
```

Comment:
Please use the `/atm-admin/property/supported-countries` endpoint to fetch such data from